CustomScript/manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.OSTCExtensions</ProviderNameSpace>
+  <Type>CustomScriptForLinux</Type>
+  <Version></Version>
+  <Label>Microsoft Azure Custom Script Extension for Linux IaaS</Label>
```

Comment:
We don't publicly use word `IaaS` in our branding. I suggest using `Virtual Machines`